tests/core/test_arg_parser.py

```
@@ -0,0 +1,153 @@
+            mock_exit.assert_called_once_with(1)
+
+    @patch("nettacker.core.arg_parser.ArgParser.load_graphs")
+    @patch("nettacker.core.arg_parser.ArgParser.load_languages")
+    @patch("nettacker.core.arg_parser.ArgParser.load_modules")
+    @patch("nettacker.core.arg_parser.ArgParser.load_profiles")
+    @patch("nettacker.core.arg_parser.ArgParser.add_arguments")
+    @patch("nettacker.core.arg_parser.ArgParser.parse_arguments")
+    def test_modules_extra_args_valid_format(
+        self,
+        mock_parse_args,
+        mock_add_args,
```

Ruff (0.14.14)

[warning] Unused method argument: `mock_add_args`

(ARG002)